.github/workflows/ci.yml

```diff
@@ -20,3 +20,4 @@ jobs:
     - run: nix-build
     - run: nix flake check
     - run: nix-shell --command 'cabal run pluto -- run examples/hello.pluto'
+    - run: nix-shell --command 'cabal test'
```

Comment:
@morganthomas The CI is not running for this PR. Maybe it is something to do with me opening the PR from my own fork (`srid/pluto`), rather than from a branch in this repo (which I don't have write access to).